src/template.lib.php

```diff
@@ -81,6 +81,20 @@ private static function replacePseudoVars($content = '', $params = array())
             }
         }
 
+        global $locale;
+
+        require_once __DIR__ . '/strings.php';
```

Comment:
Seems like this require_once is not necessary

Reply:
@yaelcampo oh good call, yeah no reason to load